File: WindowsFormsApp1/WOTR/Form2.vb
Imports System.Data.SqlClient

Public Class FormConnexion
    'Connexion au Serveur MySQL et à la BD
    Public ConnexionWOTR As SqlConnection = New SqlConnection("Data Source=Rose\SIOI;Initial Catalog = AEJT_WOTR;Integrated Security =True")

    Private Sub FormConnexion_Load(sender As Object, e As EventArgs) Handles MyBase.Load
    End Sub

    Private Sub buttonAnnuler_Click(sender As Object, e As EventArgs) Handles buttonAnnuler.Click
        textBoxCode.Text = ""
        textBoxID.Text = ""
        textBoxMail.Text = ""
        textBoxMDP.Text = ""
    End Sub

    Private Sub Valider_Click(sender As Object, e As EventArgs) Handles Valider.Click
        ConnexionWOTR.Open()
        ' Instanciation et définition de la commmande
        Dim Commande As SqlCommand = New SqlCommand("SELECT idClie, pwdClie, codeAccesClie, mailClie FROM client where idClie=01", ConnexionWOTR)
        Dim Lecteur As SqlDataReader = Commande.ExecuteReader()

        If (Lecteur.Read()) Then
            Dim count = 0
            'MessageBox.Show(Lecteur("idClie") + " " + Lecteur("pwdClie") + " " + Lecteur("codeAccesClie") + " " + Lecteur("mailClie"))
            If (textBoxID.Text = Lecteur("idClie") And textBoxMDP.Text = Lecteur("pwdClie") And textBoxMail.Text = Lecteur("mailClie") And textBoxCode.Text = Lecteur("codeAccesClie")) Then
                FormCode.Show()
            End If
        End If
        ConnexionWOTR.Close()
    End Sub

End Class

File: WindowsFormsApp1/WindowsFormsApp1/Form2.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Data.SqlClient;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WindowsFormsApp1
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
         }

        private void buttonAnnuler_Click(object sender, EventArgs e)
        {
            textBoxID.Text = "";
            textBoxCode.Text = "";
            textBoxMail.Text = "";
            textBoxMDP.Text = "";
        }
    }
}
